src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,150 @@
+import json
+
+from py42.clients import BaseClient
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    TODO Documentation here
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
+
+    def add(self, user_id):
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/highriskemployee/add"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def set_alerts_enabled(self, enabled=True):
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
+            "alertsEnabled": enabled,
+        }
+        resource = u"/highriskemployee/setalertstate"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def remove(self, user_id):
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/highriskemployee/remove"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get(self, user_id):
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/user/getbyid"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def search(
```

Comment:
We would want this to be a generator that automatically handles paging, similar to the `get_all` functions in `user`, `org`, and `device`. In fact, this should probably be called `get_all` to be consistent.

Reply:
ok, I will revert this change.

Reply:
Done